data-transform.qmd

```diff
@@ -535,7 +535,7 @@ daily_flights <- daily |>
   )
 ```
 
-If you're happy with this behavior, you can explicitly request it in order to suppress the message:
+If you're not happy with this behavior, you can explicitly request it in order to suppress the message:
```

Comment:
I don't this should be negated; specifying `drop_last` means that you're happy with the default behaviour, you just don't want the message.

Reply:
Makes sense :-)